src/long-to-linked-pe.cpp

```diff
@@ -0,0 +1,318 @@
+#include "btllib/seq_reader.hpp"
+#include "btllib/status.hpp"
+#include "btllib/util.hpp"
+
+#include <cassert>
+#include <condition_variable>
+#include <cstdio>
+#include <cstdlib>
+#include <cstring>
+#include <fstream>
+#include <functional>
+#include <getopt.h>
+#include <iostream>
+#include <limits>
+#include <math.h>
+#include <memory>
+#include <mutex>
+#include <queue>
+#include <sstream>
+#include <string>
+#include <vector>
+
+const static std::string PROGNAME = "long-to-linked-pe";
+const static std::string VERSION = "v1.0";
+const static size_t MAX_THREADS = 6;
+
+static void
+print_error_msg(const std::string& msg)
+{
+	std::cerr << PROGNAME << ' ' << VERSION << ": " << msg << std::endl;
+}
+
+static void
+print_usage()
+{
+	std::cerr
+	    << "Usage: Split long reads into paired-end pseudo-linked reads." << PROGNAME
+	    << " -l L -g G [--fasta -s -d -p P -c C -m M -t T -f FILE -b B --bx / (--bx-only)]  READS "
+	       "\n\n"
+	       "  -l L        Use L as simulated read length size.\n"
+	       "  -g G        Use G as Genome size (bp) for calculating tigmint-long span "
+	       "parameter as an integer or in scientific notation (e.g. '3e9').\n"
+	       "  --fasta     Output in fasta format.\n"
+	       "  -f FILE     Write estimated parameter to FILE. [tigmint-long.params.tsv]\n"
+	       "  -s          Calculate span parameter for tigmint-long automatically.\n"
+	       "  -c C        Use 'C * sequence coverage' to estimate span parameter. [0.25]\n"
+	       "  -d          Calculate dist parameter for tigmint-long automatically.\n"
+	       "  -p P        Use P percentile to estimate dist parameter. [50].\n"
+	       "  -m M        M minimum read length for a read to be considered a molecule. [2000].\n"
+	       "  -t T        Use T number of threads (max 6) per input file. [6]\n"
+	       "  -b B        Write barcode multiplicity to B. [tigmint-long.params.tsv].\n"

```

Comment:
I think the default should be different from `-f` -- ARCS would fail if you give it a barcode multiplicity file with those Tigmint parameter lines

Reply:
Oh I think that you actually have the default as `barcode_multiplicity.tsv`?